repos.go

```diff
@@ -69,6 +69,68 @@ var recentReleasesQuery struct {
 	} `graphql:"user(login:$username)"`
 }
 
+/*
+Order by stars
+{
+  organization(login: "charmbracelet") {
+    login
+    repositories(
+      first: 3
+      privacy: PUBLIC
+      orderBy: {field: STARGAZERS, direction: DESC}
+    ) {
+      totalCount
+      edges {
+        cursor
+        node {
+          nameWithOwner
+        }
+      }
+    }
+  }
+}*/
+
+var popularReposOrgQuery struct {
```

Comment:
```suggestion
var popularOrgReposQuery struct {
```

It sounds better to me, but I might be wrong